src/zenml/cli/utils.py

```diff
@@ -148,26 +147,23 @@ def print_table(obj: List[Dict[str, Any]], **columns: table.Column) -> None:
     column_names = [columns.get(key, key.upper()) for key in column_keys]
     rich_table = table.Table(box=box.HEAVY_EDGE, show_lines=True)
     for col_name in column_names:
-        rich_table.add_column(str(col_name), overflow="fold")
-
+        if isinstance(col_name, str):
+            rich_table.add_column(str(col_name), overflow="fold")
+        else:
+            rich_table.add_column(str(col_name.header).upper(), overflow="fold")
     for dict_ in obj:
         values = []
         for key in column_keys:
             if key is None:
                 values.append(None)
             else:
                 value = str(dict_.get(key))
-                if value == "":
+                if value in {"", "None"}:
                     value = " "
```

Comment:
```python
value = str(dict_.get(key) or " ")
```

I'd prefer having it like this to avoid this check for the `"None"` string, wdyt?

Reply:
That's a nice way to handle it!